target/ppc/translate/storage-ctrl-impl.c.inc

```diff
@@ -91,6 +91,21 @@ static bool trans_SLBMTE(DisasContext *ctx, arg_SLBMTE *a)
     return true;
 }
 
+static bool trans_SLBMFEV(DisasContext *ctx, arg_SLBMFEV *a)
+{
+    REQUIRE_64BIT(ctx);
+    REQUIRE_INSNS_FLAGS(ctx, SEGMENT_64B);
+    REQUIRE_SV(ctx);
+
+#if !defined(CONFIG_USER_ONLY) && defined(TARGET_PPC64)
+    gen_helper_load_slb_vsid(cpu_gpr[a->rt], cpu_env,
+                             cpu_gpr[a->rb]);
```

Comment:
ditto

Reply:
Fixed in 281027ddba4f6dbacb33a75f011f90cf3dfe4e4d